src/Core_Command.php

```diff
@@ -116,6 +116,12 @@ public function check_update( $_, $assoc_args ) {
 	 * [--version=<version>]
 	 * : Select which version you want to download. Accepts a version number, 'latest' or 'nightly'.
 	 *
+	 * [--retry=<retry>]
+	 * : Number of retries in case of failure. Default is 0.
```

Comment:
I think by default it can be 3. Since this retry functionality is in place, a default would be better.

Reply:
I am not sure about this part though. If it's just user opt-in, then makes sense to keep it 0?